developers/weaviate/api/graphql/get.md

```diff
@@ -40,6 +40,190 @@ The above query will result in something like the following:
 The order of object retrieval is not guaranteed in a `Get` query without any search parameters or filters. Accordingly, such a `Get` query is not suitable for any substantive object retrieval strategy.
 :::
 
+### groupBy argument
+
+You can use a groupBy argument to retrieve groups of objects from Weaviate. This functionality offers the advantage of maintaining granular search results by searching through detailed or segmented objects (e.g. chunks of documents), while also enabling you to step back and view the broader context of the objects (e.g. documents as a whole).
+
+The `groupBy{}` argument is structured as follows for the `Get{}` function:
+
+```graphql
+{
+  Get{
+    <Class>(
+      <vectorSearchParameter>  # e.g. nearVector, nearObject, nearText
+      groupBy:{
+        path: [<propertyName>]  # Properties to group by
```

Comment:
it's a path to one property always, this is an array bc in the future group by could support grouping by cross-refs specific field for example `["ofDocument", "Document", "title"]` now it's only possible to group by a property or just reference `["ofDocument"]` only

Reply:
Thanks!